.github/workflows/scripts/e2e-verify.common.sh

```diff
@@ -67,8 +67,10 @@ e2e_verify_common_all_v1() {
 # Verifies common fields of the SLSA v1.0 predicate buildDefinition.
 # $1: the predicate content
 e2e_verify_common_buildDefinition_v1() {
+    echo "pred: $1"
     # This does not include buildType since it is not common to all.
     e2e_verify_predicate_v1_buildDefinition_externalParameters_source "$1" "{\"uri\":\"git+https://github.com/$GITHUB_REPOSITORY@$GITHUB_REF\",\"digest\":{\"sha1\":\"$GITHUB_SHA\"}}"
+    e2e_verify_predicate_v1_buildDefinition_externalParameters_inputs "$1" '{"name1":"value1","name2":"***","name3":"value3","name4":"***","name5":"value5","name6":"***","private-repository":true}'
```

Comment:
This is common build definition verification for all v1.0 provenance - the docker-based ones don't include inputs like this, so it'll make that job fail

I'd move this line to the verify-setup-token.sh / verify-verified-token.sh scripts

Reply:
verify-setup-token does not mask inputs. I'll move this line to the verify-verified-token.sh instead. Works?